test/common.jl

```diff
@@ -235,11 +241,11 @@ using StaticArrays
         function oop_ṗ(p, q, params, t)
             dp1 = -q[1] * (1 + 2q[2])
             dp2 = -q[2] - (q[1]^2 - q[2]^2)
-            @SVector [dp1, dp2]
```

Comment:
Why did you remove `SVector` initial conditions?

Reply:
Thanks for checking this! I removed them temporarily because it doesn't work with latest commit 😅 since `setindex!` is not defined for `SArray{S,Taylor1}`, I'm trying to fix that right now, just pushed this to see how travis reacted

Reply:
I'll add them again as soon as I fix this

Reply:
They are back now... not the prettiest solution, but it was the best compromise until we properly support oop problems in TaylorIntegration.jl